client/src/app.config.js

```diff
@@ -0,0 +1,8 @@
+export default ({ config }) => ({
```

Comment:
This file should not be commited to git. Everyone of us needs this to be different.